metals/src/main/scala/scala/meta/internal/metals/doctor/WorkspaceDoctor.scala

```diff
@@ -0,0 +1,155 @@
+package scala.meta.internal.metals.doctor
+
+import java.util.concurrent.atomic.AtomicBoolean
+
+import scala.meta.internal.metals.BuildInfo
+import scala.meta.internal.metals.ClientCommands
+import scala.meta.internal.metals.ClientConfiguration
+import scala.meta.internal.metals.HtmlBuilder
+import scala.meta.internal.metals.MetalsHttpServer
+import scala.meta.internal.metals.ParametrizedCommand
+import scala.meta.internal.metals.Urls
+import scala.meta.internal.metals.clients.language.MetalsLanguageClient
+import scala.meta.internal.metals.config.DoctorFormat
+
+class WorkspaceDoctor(
```

Comment:
```suggestion
class HeadDoctor(
```
I think that would be "ordynator" in polish :D 

We are getting a full Metals Clinic here